.rubocop.yml

```diff
@@ -0,0 +1,4 @@
+AllCops:
+  Include:
+    - 'lib/statsample-glm/glm/formula.rb'
+    - 'lib/statsample-glm/glm/regression.rb'
```

Comment:
@zverok I'm trying to include rubocop for the code I will be writing. The Rubocop gives many offences in the current code. I'm not planning to solve them at this time. So, is there a way to just include these two files (the two files above), so that Rubocop only runs for these two files and none except these two? I tried just an obvious thing and it isn't working. Rubocop still runs for all other files.

Thanks